### PR TITLE
feat: new automated emails on pro platform

### LINKED ISSUE #10463: When adding a new user to an organisation, send automatic email to the new user to inform him/her
In the Pro platform, it is possible for a user to invite another user into an organisation, however no email is sent to the new user. 

In my opinion, it would be nice to generate this email: 

----
Subject: Welcome to your Open Food Facts workspace!

Dear **[Recipient's Name]**,

We are excited to inform you that you have been added to the workspace of **org-xxx** on the Open Food Facts Pro platform!

To get started, you can access your workspace **here [insert Pro platform link including a direction to the right org]**.

If you need any assistance, feel free to consult [our guide](https://blog.openfoodfacts.org/en/EN-Pro-Plateform-User-Guide.pdf).

Welcome aboard! We're thrilled to have you with us.

Best,

The Open Food Facts Team

### PR DESCRIPTION
**Pull request by 4nt0ineB**
### What

- When an org is rejected by an admin, an email is sent to its main contact
- En email is sent to the user when is added to an org

### Related issue(s) and discussion

- Fixes #10463

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
Thank you!

File: lib/ProductOpener/Users.pm [hunk 1]
```diff
@@ -1073,6 +1076,19 @@ sub add_users_to_org_by_admin ($org_id, $email_list) {
 			# Add the user to the organization
 			add_user_to_org($org_id, $user_id, ["members"]);
 			push @emails_added, $email;
+
+			# send user an email
+			my $user_ref = retrieve_user($user_id);
+			my $template_name = "user_added_to_org.tt.html";
```
Comment (stephanegigandet) [inline lib/ProductOpener/Users.pm:1082]: One thing I was wondering is whether we want that email to be sent by Product Opener or by Odoo?

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

File: lib/ProductOpener/Users.pm (hunk 1 above)
Comment (4nt0ineB) [reply lib/ProductOpener/Users.pm:1082]: Yes, now that you tell me I believe this might be sent by Odoo actually.

**Pull request closed without merging**